Makefile

```
@@ -8,8 +8,8 @@ CGO_ENABLED ?= 0
 
 TEMPORAL_ROOT := temporal
 TCTL_ROOT := tctl
-CLI_ROOT := cli
 DOCKERIZE_ROOT := dockerize
+CLI_VER := v0.12.0
```

I realize this might make things non-reproducible, but:

```suggestion
CLI_VER := $(shell gh release list -R temporalio/cli --json tagName,isLatest -q '.[]|select(.isLatest)|.tagName')
```

---

I'd rather us _explicitly_ set the version as a version change can break everything (as we found with upgrading to v0.12.0).

I don't want my build to break if I run it twice in a row just because the CLI team cut a new release. We can use the same trick we use currently to bump the version by running `sed` against the Makefile instead of updating a submodule

---

Yeah that makes perfect sense to me. Just wanted to mention in case it's useful (even somewhere else).